app/assets/javascripts/controllers/container_dashboard/container_dashboard_controller.js

```diff
@@ -114,64 +114,41 @@ angular.module('containerDashboard', ['ui.bootstrap', 'patternfly', 'patternfly.
           dashboardUtilsFactory.updateStatus($scope.objectStatus.routes, data.status.routes);
 
           // Node utilization donut
-          $scope.cpuUsageData = data.ems_utilization.cpu;
-          if (!$scope.cpuUsageData) {
-            $scope.cpuUsageData = { dataAvailable: false }
-          }
-
-          $scope.memoryUsageData = data.ems_utilization.mem;
-          if (!$scope.memoryUsageData) {
-            $scope.memoryUsageData = { dataAvailable: false }
-          }
+          $scope.cpuUsageData = chartsMixin.processUtilizationData(data.ems_utilization.cpu,
+                                                                   "dates",
```

Comment:
OK, then I think this is good .. (though my ..dislike of `<table>` from #6145 still stands)